airflow/contrib/operators/dataproc_operator.py

```diff
@@ -93,6 +93,9 @@ class DataprocClusterCreateOperator(BaseOperator):
     :param subnetwork_uri: The subnetwork uri to be used for machine communication,
         cannot be specified with network_uri
     :type subnetwork_uri: string
+    :param internal_ip_only: Restriction only available if subnetwork_uri is defined,
```

Comment:
Can you add more details over here i.e. when would you use this parameter?

You can use the details over here:
https://cloud.google.com/dataproc/docs/reference/rest/v1/projects.regions.clusters#gceclusterconfig

>Optional. If true, all instances in the cluster will only have internal IP addresses. By default, clusters are not restricted to internal IP addresses, and will have ephemeral external IP addresses assigned to each instance. This internalIpOnly restriction can only be enabled for subnetwork enabled networks, and all off-cluster dependencies must be configured to be accessible without external IP addresses.